openwisp_network_topology/tests/test_api.py

```diff
@@ -90,12 +92,12 @@ def test_detail(self):
     def test_list_unpublished(self):
         self._unpublish()
         response = self.client.get(self.list_url)
-        self.assertEqual(len(response.data['collection']), 0)
+        self.assertEqual(len(response.data['collection']), 1)
 
     def test_detail_unpublished(self):
         self._unpublish()
         response = self.client.get(self.detail_url)
-        self.assertEqual(response.status_code, 404)
+        self.assertEqual(response.status_code, 200)
```

Comment:
Doesn't this bring changes to UI has well, changing the behavior of the UI to show unpublished ones as well. (If I am undetstanding the change views.py correctly!)

Reply:
@atb00ker by `UI` do you mean the visualizer, the visualiser uses separate views. If I filter it then again it will not be possible to re-publish it, but in the admin panel, we can publish/un-publish. what would be good, should I filter it??

Reply:
But, once we filter it out , we will not be able to republish it again with the endpoint

Reply:
Also here it would be better to maintain this behavior and allow users of the API to show also unpublished items by adding the GET parameter.